### PR TITLE
Issue/2619 - Link/Button Tile background property

### PR DESCRIPTION
**Pull request by tatjana-gartner2**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://opensource.porsche.com/docs/cla)

#### References

- Preview: https://designsystem.porsche.com/issue/.../

#### Scope

Short description of the pull request scope

#### Resolved Issue

Resolves [IssueNumber]

### CODE REVIEW

**Review by o-hook — APPROVED** [review]

File: packages/components/src/components/button-tile/button-tile-styles.spec.ts [hunk 1]
```diff
@@ -2,24 +2,25 @@ import { getComponentCss } from './button-tile-styles';
...
       { base: 'semi-bold', xs: 'regular', s: 'semi-bold', m: 'regular', l: 'semi-bold', xl: 'regular' },
+      'light',
       'top',
       { base: true, xs: false, s: true, m: false, l: true, xl: false },
       true,
       false,
     ],
   ])(
-    'should return correct css for isDisabledOrLoading: %s, aspectRatio: %j, size: %j, weight: %j, align: %s, isCompact: %j, hasGradient: %s and isDisabled: %s',
+    'should return correct css for isDisabledOrLoading: %s, aspectRatio: %j, size: %j, weight: %j, background: %j, align: %s, isCompact: %j, hasGradient: %s and isDisabled: %s',
```
Comment (denyo) [inline packages/components/src/components/button-tile/button-tile-styles.spec.ts:23]: I think only bp customizable ones are interpolated with `%j` or what's the pattern?

File: packages/components/src/styles/tile/button-link-tile-styles.ts [hunk 2]
```diff
@@ -44,6 +46,9 @@ export const getButtonLinkTileStyles = (
               fontWeight: getFontWeight(w === 'semibold' ? 'semi-bold' : w), // mapping of the deprecated weight semibold
             }))
           ),
+          ...buildResponsiveStyles(background, (b: Theme) => ({
+            color: getThemedColors(b === 'dark' ? 'dark' : 'light').primaryColor,
```
Comment (denyo) [inline packages/components/src/styles/tile/button-link-tile-styles.ts:50]: Please use `isDark(theme)` utility.

File: packages/components/src/styles/tile/button-link-tile-styles.ts [hunk 3]
```diff
@@ -54,6 +59,7 @@ export const getButtonLinkTileStyles = (
           : `${spacingFluidLarge} ${spacingFluidMedium} ${spacingFluidMedium}`,
         ...mergeDeep(
           hasGradient &&
+            background === 'dark' &&
```
Comment (denyo) [inline packages/components/src/styles/tile/button-link-tile-styles.ts:62]: Here too.

File: packages/components/src/components/link-tile/link-tile-styles.spec.ts [hunk 4]
```diff
@@ -2,21 +2,22 @@ import { getComponentCss } from './link-tile-styles';
...
       { base: 'inherit', xs: 'default', s: 'inherit', m: 'default', l: 'inherit', xl: 'default' },
       { base: 'semi-bold', xs: 'regular', s: 'semi-bold', m: 'regular', l: 'semi-bold', xl: 'regular' },
+      'light',
       'top',
       { base: true, xs: false, s: true, m: false, l: true, xl: false },
       true,
     ],
   ])(
-    'should return correct css for aspectRatio: %j, size: %j, weight: %j, align: %s, isCompact: %j and hasGradient: %s',
+    'should return correct css for aspectRatio: %j, size: %j, weight: %j, background: %j, align: %s, isCompact: %j and hasGradient: %s',
```
Comment (denyo) [inline packages/components/src/components/link-tile/link-tile-styles.spec.ts:20]: I think only bp customizable ones are interpolated with `%j` or what's the pattern?

File: packages/components/src/styles/tile/button-link-tile-styles.ts [hunk 5]
```diff
@@ -44,6 +47,9 @@ export const getButtonLinkTileStyles = (
               fontWeight: getFontWeight(w === 'semibold' ? 'semi-bold' : w), // mapping of the deprecated weight semibold
             }))
           ),
+          ...buildResponsiveStyles(background, (b: Theme) => ({
+            color: getThemedColors(isThemeDark(b) ? 'dark' : 'light').primaryColor,
```
Comment (denyo) [inline packages/components/src/styles/tile/button-link-tile-styles.ts:51]: Wouldn't this have the same effect?
```suggestion
            color: getThemedColors(b).primaryColor,
```

File: packages/components/src/styles/tile/button-link-tile-styles.ts [hunk 6]
```diff
@@ -14,6 +14,8 @@ import {
 import { buildResponsiveStyles, mergeDeep } from '../../utils';
 import { getFontWeight } from '../font-weight-styles';
 import { getTileBaseStyles } from './tile-base-styles';
+import { getThemedColors } from '../../styles';
+import { isThemeDark } from '../../utils/theme';
```
Comment (denyo) [inline packages/components/src/styles/tile/button-link-tile-styles.ts:18]: Please combine imports, it's the 3rd one from utils now.

**Pull request merged**